src/components/MovieDetails/MovieDetails.js

```diff
@@ -2,11 +2,68 @@ import React from 'react';
 import './MovieDetails.scss';
 import { trailers } from '../../trailers';
 
-const MovieDetails = ({ location }) => {
+import { connect } from 'react-redux';
```

Comment:
Non-blocking, but traditionally imports should be ordered with third-party at the top, then local imports, then stylesheets at the bottom.

**Example**
```javascript
import React from 'react';
import { trailers } from '../../trailers';
import './MovieDetails.scss';
```